cmd/create/cluster/cmd.go

```diff
@@ -493,37 +517,115 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
-	// Compute nodes:
-	computeNodes := args.computeNodes
-	// Compute node requirements for multi-AZ clusters are higher
-	if multiAZ && !cmd.Flags().Changed("compute-nodes") {
-		computeNodes = 3
-	}
-	if interactive.Enabled() {
-		computeNodes, err = interactive.GetInt(interactive.Input{
-			Question: "Compute nodes",
-			Help:     cmd.Flags().Lookup("compute-nodes").Usage,
-			Default:  computeNodes,
+	// Autoscaling
+	isAutoscalingSet := cmd.Flags().Changed("enable-autoscaling")
+	autoscaling := args.autoscalingEnabled
+	if !autoscaling && !isAutoscalingSet && interactive.Enabled() {
+		autoscaling, err = interactive.GetBool(interactive.Input{
+			Question: "Enable autoscaling",
+			Help:     cmd.Flags().Lookup("enable-autoscaling").Usage,
+			Default:  autoscaling,
+			Required: false,
 		})
 		if err != nil {
-			reporter.Errorf("Expected a valid number of compute nodes: %s", err)
+			reporter.Errorf("Expected a valid value for enable-autoscaling: %s", err)
 			os.Exit(1)
 		}
 	}
-	if multiAZ {
-		if computeNodes < 3 {
-			reporter.Errorf("The number of compute nodes needs to be at least 3")
+
+	isMinReplicasSet := cmd.Flags().Changed("min-replicas")
+	isMaxReplicasSet := cmd.Flags().Changed("max-replicas")
+
+	minReplicas := args.minReplicas
+	maxReplicas := args.maxReplicas
+	if autoscaling {
+
+		// if the user set compute-nodes and enabled autoscaling
+		if cmd.Flags().Changed("compute-nodes") {
+			reporter.Errorf("Compute-nodes can't be set when autoscaling is enabled")
 			os.Exit(1)
 		}
-		if computeNodes%3 != 0 {
+
+		if interactive.Enabled() || !isMinReplicasSet {
+			minReplicas, err = interactive.GetInt(interactive.Input{
+				Question: "Min replicas",
+				Help:     cmd.Flags().Lookup("min-replicas").Usage,
+				Default:  minReplicas,
+				Required: true,
+			})
+			if err != nil {
+				reporter.Errorf("Expected a valid number of min replicas: %s", err)
+				os.Exit(1)
+			}
+		}
+		if interactive.Enabled() || !isMaxReplicasSet {
+			maxReplicas, err = interactive.GetInt(interactive.Input{
+				Question: "Max replicas",
+				Help:     cmd.Flags().Lookup("max-replicas").Usage,
+				Default:  maxReplicas,
+				Required: true,
+			})
+			if err != nil {
+				reporter.Errorf("Expected a valid number of max replicas: %s", err)
+				os.Exit(1)
+			}
+		}
+
+		if minReplicas < 2 {
```

Comment:
This number should be dependent on the multi-az flag too

Reply:
Done

Reply:
fixed